src/main/java/ch/njol/util/coll/CollectionUtils.java

```diff
@@ -103,6 +96,20 @@ public static <T> boolean containsAll(final @Nullable T[] array, final @Nullable
 		return true;
 	}
 	
+	public static <T> boolean check(T @Nullable [] array, Predicate<@UnknownNullability T> predicate, boolean and) {
```

Comment:
any reason to not use Java's Predicate? this would be the only usage of Skript's Predicate and I was considering a PR to deprecate it since it's unused

Reply:
Whoops, it was an accident